RELEASE-NOTES.md

```diff
@@ -0,0 +1,128 @@
+# ResearchKit Release Notes
+
+## ResearchKit 1.2 Release Notes
+
+*ResearchKit 1.2* supports *iOS* and requires *Xcode 7.0* or newer. The minimum supported *Base SDK* is *8.0*.
+
+- **New Active Tasks**
+
+ - **Tower of Hanoi Task**
+
+    *Contributed by [coxy1989](https://github.com/coxy1989).*
+
+    The *[Tower of Hanoi](https://en.wikipedia.org/wiki/Tower_of_Hanoi#Applications) task* is frequently used in psychological research on problem solving.
+
+    It is a mathematical puzzle consisting of three rods and a number of disks of different sizes which can slide onto any rod. The puzzle starts with the disks in a stack in ascending order of size on one rod (the smallest at the top).
+
+    The objective of the puzzle is to move the entire stack to another rod, obeying the following rules:
+
+    1. Only one disk can be moved at a time.
+    2. Each move consists of taking the upper disk from one of the stacks and placing it on top of another stack.
+    3. No disk may be placed on top of a smaller disk.
+
+ - **Paced Serial Addition Test (PSAT) Task**
+
+    *Contributed by [Julien Therier](https://github.com/julientherier).*
+
+    The *PSAT task* provides support for both the *Paced Auditory Serial Addition Test (PASAT)* and the *Paced Visual Serial Addition Test (PVSAT)* versions.
+
+    The *[PASAT](https://en.wikipedia.org/wiki/Paced_Auditory_Serial_Addition_Test)* is a neuropsychological test used to assess capacity and rate of information processing and sustained and divided attention.
+
+    This task is documented in the scientific literature ([[Fos et al., 2000]](http://www.ncbi.nlm.nih.gov/pubmed/11125707), [[Nagels et al., 2005]](http://www.ncbi.nlm.nih.gov/pubmed/15823678)) as a measure of the [*Multiple Sclerosis Functional Score*](http://www.nationalmssociety.org/For-Professionals/Researchers/Resources-for-Researchers/Clinical-Study-Measures/Multiple-Sclerosis-Functional-Composite-(MSFC).
+
+    It generates a series of single digits (for example, 60), at the specific frequency (for example, one new digit every 2 or 3 seconds). The user must add the newly presented digit to the one prior to it.
```

Comment:
From our internal edit:
Please replace `It generates` with `This task generates`.